style.css

```diff
@@ -66,11 +66,48 @@ body {
 }
 
 .menu__item {
-    padding: 0 13px;
+    position: relative;
+    margin-right: 26px;
+    font: 500 18px/21px 'Roboto', Arial, Helvetica, sans-serif;
+    color: #000;
+    text-decoration: none;
+    transition: opacity .5s;
 }
 
 .menu__item:last-child{
-    padding-right: 0;
+    margin-right: 0;
+}
+
+.menu__item:hover{
+    opacity: .5;
+}
+
+.menu__item:after {
+    position: absolute;
+    display: block;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/kastykina_app.creation/pullRequest?prid=6252582)